### PR TITLE
Ch setup houndci eslint 158911897

### PR DESCRIPTION
**Pull request by jherey**
#### What does this PR do?
- Adds eslint configuration and HoundCI

#### Description of Task to be completed?
- Create .hound.yml file to root of application.
- Add repository to hound

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#158911897

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,6 +93,7 @@
     "react-measure": "^2.0.0",
     "react-nouislider": "^2.0.0",
     "react-onclickoutside": "^6.4.0",
+    "react-portal": "^4.1.5",
```
Comment (fortunee) [inline package.json]: @jherey What's this package for?

**Review by fortunee — COMMENTED** [review]
@jherey Please check out my comment on your PR

File: package.json (hunk 1 above)
Comment (jherey) [reply package.json]: @fortunee I had issues setting up the project, so I installed it so as to render the DOM but forgot to uninstall when I figured it out. I'll remove and make a new PR.

Comment (fortunee) [reply package.json]: You don't have to make a new PR. Just remove and push @jherey

Comment (jherey) [reply package.json]: Ok, I'll do that.

Comment (jherey) [reply package.json]: Done @fortunee

**Pull request merged**